### ISSUE TITLE
Screen artifacts on top bar

### ISSUE THREAD

**Issue by LEMUSADR000** [jamesblasco/modal_bottom_sheet#129]
The top bar of the page has screen white artifact-like pixels on left and right corners. Here are some images:  
  
  
1. What it looks like on my project
![Screenshot_20201221-110302](https://user-images.githubusercontent.com/16787847/102813350-7f42f680-437d-11eb-912b-f03f1ee04d2e.png)  
  
  
2. What it looks like on project 'Modal with PageView' sample
![Screenshot_20201221-110350](https://user-images.githubusercontent.com/16787847/102813387-91bd3000-437d-11eb-9fac-f400c3b784d9.png)
  
  
Closer shot for clarity:
![Screenshot_20201221-110350](https://user-images.githubusercontent.com/16787847/102813906-861e3900-437e-11eb-819b-62b4f92e0a73.png)
  
  

Steps to reproduce:

- Download [example project](https://github.com/jamesblasco/modal_bottom_sheet/tree/master/example)
- Run application on Android device (I used Moto g7) 
- Click on 'Modal with PageView' 

So far I've tried tweaking shape and topRadius but they don't seem to change the presence of these artifacts. Do you know how to fix this? If not, do you know what could be the cause so I could possibly fix it myself and submit an MR?

**Comment by jamesblasco** [comment]
Yes, I think it is the added border that it should not be there. I will remove it in the next version

**Comment by LEMUSADR000** [comment]
This is a great looking package, thanks for all of your work on this!

**Comment by Zabadam** [comment]
In one of my own apps, I have a Clipper with the same shape as an underlying Container. I still wanted the Border to match, but I ended up reducing the radius of my clipper by 0.5 compared to the container for a cleaner edge. Probably irrelevant in this situation, but something to have in mind.

**Comment by jamesblasco** [comment]
Hello, there are two errors in the BarBottomSheetRight now. 
First there was a unneeded border that has been removed,
Second, the clips had the behavior Clip.hardEdge by default. Clip.antiAlias is much smoother, but it is slower. I will try that in my projects, and if it goes well I will make the change soon. Feel free to try it by adding the param clipBehaviour

**Comment by LEMUSADR000** [comment]
Changing `clipBehavior` didn't seem to make a noticeable difference in the behavior I was seeing. I'm still getting the white artifacts.

**Comment by armandojimenez** [comment]
Any way to deal with this?

**Comment by LEMUSADR000** [comment]
I couldn't seem to find a legit workaround for it so I just made any usages of this widget have white bar colors for now. The shape is still messed up, though. It doesn't seem to have a perfect radius.

**Comment by Zabadam** [comment]
I have never personally experienced this visual bug when using this package, but I also use bespoke styled objects for each portion of the [SlidingSheet], i.e. my [SlidingSheet.headerBuilder] returns a [SlidingSheetHeader].

I do not use any of the built-in borderRadius parameters. Are you? If so, you could instead change the borderRadius of your `header` manually to see if that is the culprit.
